oss/stb/stb_rect_pack.h

```diff
@@ -466,7 +466,7 @@ static stbrp__findresult stbrp__skyline_pack_rectangle(stbrp_context *context, i
 
    // insert the new node into the right starting point, and
    // let 'cur' point to the remaining nodes needing to be
-   // stiched back in
+   // stitched back in
```

Comment:
Can you revert the changes to the `/dep` and `/oss` directories? Those are external source files we've vendored.